src/server/repositories/UserRepository.ts

```diff
@@ -49,7 +50,12 @@ export class UserRepository implements UserRepositoryInterface {
   public findOrCreateWithEmail: (email: string) => Promise<StorableUser> = (
     email,
   ) => {
-    return User.findOrCreate({ where: { email } }).then(([user, _]) => user)
+    return User.findOrCreate({ where: { email } }).then(([user, created]) => {
+      if (created) {
+        dogstatsd.increment('user.new', 1, 1)
```

Comment:
is it correct to increase the count here? 
Since the function is `findOrCreateWithEmail`, we might potentially run into existing user

Reply:
yep it should be correct: the `created` in [`findOrCreate`](https://sequelize.org/docs/v6/core-concepts/model-querying-finders/#findorcreate) is a boolean telling us if a new user's been created, so the metric will increment only if the user is new

trying it out on staging, I also haven't observed any metric increments when logging in with my own existing email, so it matches what we would expect!